### DISCUSSION TITLE
Why does my application log show 'New conversation' here？

### DISCUSSION THREAD

**Discussion by LpLBraveHeart** [langgenius/dify#10310] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

![Snipaste_2024-11-05_17-55-01](https://github.com/user-attachments/assets/33625796-acb5-4a04-87e3-ab58e4933650)

**Comment by if0then1** [comment]
Is this solved? I have the same problem.

**Comment by mu-dan** [comment]
dify版本优化的问题，咋解决啊
